DOCUMENTATION.md

```diff
@@ -1,70 +1,121 @@
-# GOV.UK Notify PHP client
+_Speak to Leo Hemsted_
 
-This documentation is for developers interested in using this PHP client to integrate their government service with GOV.UK Notify.
+# PHP client documentation
 
-## Table of Contents
+This documentation is for developers interested in using the GOV.UK Notify PHP client to send emails, text messages or letters.
 
-* [Installation](#installation)
-* [Getting started](#getting-started)
-* [Send messages](#send-messages)
-* [Get the status of one message](#get-the-status-of-one-message)
-* [Get the status of all messages](#get-the-status-of-all-messages)
-* [Get a template by ID](#get-a-template-by-id)
-* [Get a template by ID and version](#get-a-template-by-id-and-version)
-* [Get all templates](#get-all-templates)
-* [Generate a preview template](#generate-a-preview-template)
-* [Get received text messages](#get-received-text-messages)
-* [Development](#development)
-* [License](#license)
+# Installation and getting started
 
-## Installation
+The Notify PHP Client is based on a [PSR-7 HTTP model](https://www.php-fig.org/psr/psr-7/) [external link]. You must select your preferred HTTP Client library. We provide installation instructions for the [Guzzle v5 and v6](http://docs.guzzlephp.org/en/stable/) and [Curl](http://php.net/manual/en/book.curl.php) [external links] clients.
 
-The Notify PHP Client can be installed with [Composer](https://getcomposer.org/). Run this command:
+## Guzzle v6
+
+The Notify PHP Client is installed with [Composer](https://getcomposer.org/).
+
+Run the following in the command line to install the client:
 
 ```sh
 composer require php-http/guzzle6-adapter alphagov/notifications-php-client
 ```
 
-### PSR-7 HTTP
+The Notify PHP Client is then available via the [autoloader](https://getcomposer.org/doc/01-basic-usage.md#autoloading) [external link].
+
+Add the following code to your application to create a new instance of the client:
 
-The Notify PHP Client is based on a PSR-7 HTTP model. You therefore need to pick your preferred HTTP Client library to use.
+```sh
+$notifyClient = new \Alphagov\Notifications\Client([
+    'apiKey' => '{your api key}',
+    'httpClient' => new \Http\Adapter\Guzzle6\Client
+]);
+```
 
-We will show examples here using the Guzzle v6 Adapter.
+To get an API key, [sign in to GOV.UK Notify](https://www.notifications.service.gov.uk/) and go to the __API integration__ page. Refer to the [API keys](#api-keys) section of this documentation for more information.
 
-Setup instructions are also available for [Curl](docs/curl-client-setup.md) and [Guzzle v5](docs/guzzle5-client-setup.md).
+## Guzzle v5
```

Comment:
there's still a lot of shared code between these things. All the text is the same - the only difference is 

`composer require php-http/guzzle5-adapter`
and
```php
'httpClient'    => new \Http\Adapter\Guzzle5\Client(
        new \GuzzleHttp\Client,
        new \Http\Message\MessageFactory\GuzzleMessageFactory
    ),
```

which change between versions. Reckon we can reduce this any more at all?

Reply:
User research has suggested that dev users want everything self-contained, so they can just refer to the relevant section, hence the duplication. Happy to chat about.